http/1.6/modules/ROOT/pages/http-authentication.adoc

```diff
@@ -180,209 +144,209 @@ The GitHub API returns your user information.
+     doc:name="HTTP Request Configuration" >
+  <http:request-connection
+     host="example.com"
+     port="8081" >
+     <http:authentication >
+        <http:ntlm-authentication username="Username" password="Password" />
+     </http:authentication>
+  </http:request-connection>
 </http:request-config>
 
 <flow name="digest_flow">
     ...
-    <http:request config-ref="HTTP_Request_configuration"
+    <http:request method="GET" doc:name="Request"
+    config-ref="HTTP_Request_configuration"
     path="test"
-    method="GET" />
+     />
 
 </flow>
 ----
 
-In this example, a GET request is sent to `+http://www.example.com/test+`, adding an "Authorization" header with the provided username and password.
+== Configure OAuth2 Authorization Code Grant Type Authentication
 
-== OAuth2 - Authorization Code
+The OAuth2 authorization code configures the OAuth 2.0 authorization code grant type. The OAuth authentication server holds the resources protected by OAuth. For example, API calls to the GitHub API can be authenticated through https://developer.github.com/v3/oauth/[GitHub server using OAuth].
 
-The OAuth2 - Authorization Code configures the OAuth 2.0 authorization code grant type. The OAuth authentication server holds the resources that are protected by OAuth. For example, calls to the GitHub API can be authenticated through https://developer.github.com/v3/oauth/[GitHub server using OAuth]. The example in this section shows you how to create Mule client application to access a protected resource, GitHub user data, on the GitHub OAuth authentication server. The example covers the following things:
+To configure the OAuth2 Authorization code grant type for the HTTP *Request* operation review the following example that shows how to create a Mule app to access a protected resource, GitHub user data, on the GitHub OAuth authentication server. The example covers how to:
 
 * Set up authorization
-* Create a Mule client app
-* Run the Mule client app
+* Create a Mule app
+* Run the Mule app
 
 This example requires that you have a GitHub account.
 
 === Set Up Authorization
 
-. Register the client application on the authentication server. The authentication server assigns a client ID and client secret to the Mule client app. The app uses these credentials later to identify itself to the authentication server. During the registration, you also provide the URL to the Mule app home page and the application callback URL.
-+
-image::authentication-in-http-requests-75e03.png[]
-+
+To set up the authorization, refer to the steps on the following diagram:
+
+image::http-authentication-diagram-1.png[Set up authorization]
+
+. Register the client application on the authentication server. +
+ The authentication server assigns a client ID and client secret to the Mule app. The app uses these credentials to identify itself to the authentication server. During the registration, provide the URL to the Mule app home page and the application callback URL.
```

Comment:
```suggestion
The authentication server assigns a client ID and client secret to the Mule app. The app uses these credentials to identify itself to the authentication server. Provide the URL to the Mule app home page and the application callback URL.
```
I wonder if this step should use substeps to document the step(s) of providing the URLs?